src/components/CalendarSelect.js

```diff
@@ -0,0 +1,50 @@
+import './css/CalendarSelect.css';
+import React from "react";
+
+function CalendarSelect() {
+    // TODO: replace this demo list, probably with global context
+    const calendars = ["Appointments", "Birthdays", "Homework"]
+
+    // react to the add calendar button being clicked
+    function addCalendar() {
+        // TODO: fill in functionality, likely open calendar creation view
+    }
+
+    // react to a calendar checkbox being changed
+    function setShowCalendar(e) {
+        // TODO: fill in functionality, likely change global context for visible calendars
+        //console.log(e.target.value + ": " + e.target.checked);
+    }
+
+    return (
+        <>
+            <div id="calSelect" className="dropdown-menu show" style={{}}>
```

Comment:
whats with the empty inline styling here?

Reply:
That's my mistake, I was experimenting with styles and forgot to remove that.